src/tokens-studio/spectrum2-non-colors/spectrum2/layout/desktop.json

```diff
@@ -395,19 +395,8 @@
       }
     }
   },
-  "component-size-difference-down": {
-    "value": "2px",
-    "type": "sizing",
-    "$extensions": {
-      "spectrum-tokens": {
-        "uuid": "9ef4a0a5-b81c-4f54-8927-286ece29a824",
-        "name": "component-size-difference-down",
-        "constant-token-duplicate": false
-      }
-    }
-  },
   "component-size-width-ratio-down": {
-    "value": "3px",
+    "value": "0.3333",
     "type": "sizing",
```

Comment:
Hi @nabuhasan @lynnhao I totally missed that this token might not be of type 'sizing'. I think, this 'ratio' token and other tokens (like `text-field-minimum-width-multiplier`) could be of type 'number'. I will scan through all the non-color tokens tomorrow and create a list of tokens, that might not be sizing tokens, if that is alright with you. Thanks Frank

Reply:
No problem, let me know where specific tokens should live and I can make the updates then! @karstens